api/src/domain/loaders/load-domain-connections-by-organizations-id.js

```diff
@@ -382,12 +382,17 @@ export const loadDomainConnectionsByOrgId =
     LET retrievedDomains = (
       ${loopString}
         FILTER domain._key IN domainKeys
+        LET claimTags = (
+          FOR v, e IN 1..1 ANY domain._id claims
+            FILTER e._from == ${orgId}
+            RETURN e.tags
+        )[0]
         ${afterTemplate}
         ${beforeTemplate}
         SORT
         ${sortByField}
         ${limitTemplate}
-        RETURN MERGE({ id: domain._key, _type: "domain" }, domain)
+        RETURN MERGE({ id: domain._key, _type: "domain", "claimTags": claimTags }, domain)
```

Comment:
Should translate the tags before returning them.

[Example](https://github.com/canada-ca/tracker/blob/80611d591e5aaddc7686838bd5b3138dec01c234/api/src/guidance-tag/loaders/load-ssl-guidance-tags.js#L8-L23)